docs/contribute/developer-guide.md

```diff
@@ -131,6 +131,17 @@ Just open APITable's root directory with IDE.
 
 
 
+## How to contribute translations?
+
+We have two ways to improve the translation of APITable:
+
+1. Modify the source code and create a PR
+2. Find the `strings` to be modified in [Crowdin](https://crowdin.com/project/apitablecode) and modify them online
```

Comment:
Join our [Crowdin](https://crowdin.com/project/apitablecode) to find the `strings` to modify